build/kind/cloud/entrypoint.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Turn on bash's job control
+set -m
+
+# Logging function to make it easier to debug
+log() {
+  echo "[INFO] $1"
+}
+
+# Check if agent key is provided
+if [ -z "$AGENT_KEY" ]; then
+  log "Please provide AGENT_KEY env var"
+  exit 1
+fi
+
+# Check if org id is provided
+if [ -z "$ORG_ID" ]; then
+  log "Please provide ORG_ID env var"
+  exit 1
+fi
+
+# Check if env id is provided
+if [ -z "$ENV_ID" ]; then
+  log "Please provide ENV_ID env var"
+  exit 1
+fi
+
+# Check if cloud url is provided
+if [ -z "$CLOUD_URL" ]; then
```

Comment:
Isn't that rather `ROOT_DOMAIN`?

Reply:
I just followed exsitng agent helm installation command  `--set testkube-api.cloud.url=agent.testkube.io:443`, will ROOT_DOMAI will be clear for user?